api/hypershift/v1beta1/nodepool_conditions.go

```diff
@@ -19,6 +19,10 @@ const (
 	// A failure here is unlikely to resolve without the changing user input.
 	NodePoolValidTuningConfigConditionType = "ValidTuningConfig"
 
+	// NodePoolValidPlatformConfigConditionType signals if the content within nodePool.spec.platform is valid.
+	// A failure here is unlikely to resolve without the changing user input.
```

Comment:
"without the changing" is that a typo we are carrying over?

Reply:
fixed 👍